README.md

```diff
@@ -1,4 +1,54 @@
 # BSC-MEV-Sentry
 
-BSC-MEV-Sentry is a proxy service within the BSC MEV architecture designed to shield the validator network.
-It functions to forward requests and pay to builders.
+BSC-MEV-Sentry serves as the proxy service for BSC MEV architecture, It has the following features:
+
+1. Forward RPC requests under mev namespace to validators/builders.
```

Comment:
It is hard to understand the `mev namespace`.. I know it is Http module.

Reply:
I mean this https://github.com/bnb-chain/bsc/blob/master/internal/ethapi/backend.go#L150

Reply:
I dont think the API user know this knowledge.

Reply:
fixed